sitrep3.Rproj

```diff
@@ -0,0 +1,17 @@
+Version: 1.0
```

Comment:
@zkamvar  is this file gonna make booboos?

Reply:
It's not going to make booboos per-se, but it should be removed before this is merged.